bin/pycbc_make_sky_grid

```diff
@@ -165,23 +165,16 @@ spher = cart_to_spher(rota)
 
 # Calculate the time delays between the Earth center
 # and each detector for each sky point
-time_delays = [
-    [
-        detectors[i].time_delay_from_earth_center(
-            spher[j][0], spher[j][1], args.trigger_time
+sky_grid = sky_grid.SkyGrid(
+        spher[:, 0], spher[:, 1], args.instruments, args.trigger_time
         )
-        for j in range(len(spher))
-    ]
-    for i in range(len(detectors))
-]
+time_delays = sky_grid.calculate_time_delays()
```

Comment:
* Remove the comments about time delays, no longer relevant.
* Fix the indentation and closing parenthesis as indicated previously.
* Remove the calculation of the time delays.